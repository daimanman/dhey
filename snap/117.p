{"C":1,"Duration":107712688,"EndTime":1551409867899758000,"Err":"","FileId":"","InputFileName":"","LonhApi":"findGpsOnlinelist","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409867792045300,"Status":1,"TaskId":117,"Type":"LONH","Url":"","Z":""}