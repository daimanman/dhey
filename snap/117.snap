{"AvgTotal":0.1027401,"Fastest":0.1027401,"Slowest":0.1027401,"Average":0.1027401,"Rps":9.733297904128962,"AvgConn":0.0478698,"AvgDNS":0,"AvgReq":0,"AvgRes":0.0009507,"AvgDelay":0.0529204,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0009507,"DelayMax":0.0529204,"DelayMin":0.0529204,"Lats":[0.1027401],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0.0009507],"DelayLats":[0.0529204],"Total":102740100,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1027401,"Count":1,"Frequentcy":1},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0},{"Mark":0.1027401,"Count":0,"Frequentcy":0}],"TaskId":117,"Duration":0.1027401}